packages/history/src/index.ts

```diff
@@ -7,6 +7,7 @@ export interface NavigateOptions {
 }
 export interface RouterHistory {
   location: HistoryLocation
+  subscribers: Set<() => void>
```

Comment:
Should we straight away use the function-getter pattern?

We always end up refactoring down the line to use this pattern anyways 😅 

```suggestion
  getSubscribers () => Set<() => void>
```

Reply:
I think this one is fine as is. It *should* be very low traffic for a long time.